.cirrus.yml

```diff
@@ -628,6 +628,26 @@ verify_test_built_images_task:
     always:
         <<: *standardlogs
 
+
+upload_snap_task:
+
```

Comment:
I'm thinking it might not be good if we force every push to every PR, to be subject to snap-package upload as a condition for testing success.  Also because rarely will these PR snap packages be generally useful.  That said, uploads after a PR merges would be useful.  It's possible for automation to distinguish between execution on a PR and on a branch.  This is easy to do by simply setting  `only_if: $CIRRUS_BRANCH == $DEST_BRANCH`.  The down-side is, that makes testing this task in this PR difficult...

Reply:
Added.